static/src/javascripts/projects/common/modules/commercial/acquisitions-copy.js

```diff
@@ -36,21 +36,21 @@ const ctaLinkSentence = (
     currencySymbol: string
 ): string => {
     if (supportFrontendLiveInUs) {
-        return `For as little as ${
+        return `<span class="contributions__highlight"> For as little as ${
```

Comment:
What do you think about `<em>` instead? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/em

Reply:
It does use `<em>` for the itallics effect, this copy gets inserted into this template https://github.com/guardian/frontend/blob/master/static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-liveblog.js

The `contributions__highlight` class just does the yellow highlighting.